spec/presenters/converted_money_presenter_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```

Comment:
@dissident I think we don't need this spec. This presenter will be tested when you test `debtors/index`